src/app/views/authentication/Authentication.tsx

```diff
@@ -1,182 +1,152 @@
-          auth: ADMIN_AUTH_URL,
-          grant: TOKEN_URL,
-        },
-        scope_delim: ' ',
-        form: false,
-      },
-    } as any);
-    hello.init({
-      msft: 'cb2d7367-7429-41c6-ab18-6ecb336139a6',
-      msft_admin_consent: 'cb2d7367-7429-41c6-ab18-6ecb336139a6',
-    }, {
-        redirect_uri: window.location.pathname,
-      });
-    hello.on('auth.login', async (auth) => {
-        let accessToken;
-        let authenticated = {...this.state.authenticated};
-        if (auth.network === 'msft') {
-          const authResponse = hello('msft').getAuthResponse();
-          accessToken = authResponse.access_token;
-          if (accessToken) {
-            authenticated.token = accessToken;
-          }
-          authenticated.status = true;
-          if (actions) {
-            actions.authenticateUser(authenticated);
-          }
-        }
-        if (accessToken) {
-          try {
-            const query = {
-              sampleURL : USER_INFO_URL,
-            };
-            const userInfo = (queryActions) ? await queryActions.runQuery(query) : null;
-            const jsonUserInfo = userInfo.response.body;
-            authenticated.user = {...{},
-                                  displayName: jsonUserInfo.displayName,
-                                  emailAddress: jsonUserInfo.mail || jsonUserInfo.userPrincipalName,
-                                  profileImageUrl: '',
-            };
-            if (actions) {
-              actions.authenticateUser(authenticated);
-              this.setState({
-                authenticated,
-              });
-            }
-            try {
-              const pictureQuery = {
-                sampleURL : USER_PICTURE_URL,
-              };
-              const userPicture = (queryActions) ? await queryActions.runQuery(pictureQuery) : null;
-              const buffer = await userPicture.response.body.arrayBuffer();
-              const blob = new Blob([buffer], { type: 'image/jpeg' });
-              const imageUrl = URL.createObjectURL(blob);
-              if (actions) {
-                authenticated = this.state.authenticated;
-                authenticated.user.profileImageUrl = imageUrl;
-                actions.authenticateUser(authenticated);
-                this.setState({
-                  authenticated,
-                  loading: false,
-                });
-              }
-            } catch (e) {
-              if (actions) {
-                authenticated = this.state.authenticated;
-                authenticated.user.profileImageUrl = '';
-                actions.authenticateUser(authenticated);
-                this.setState({
-                  authenticated,
-                  loading: false,
-                });
-              }
-            }
-          } catch (e) {
-            // tslint:disable-next-line:no-console
-            console.log(e);
-          }
-        }
-      });
+  private async getUserInfo(queryActions: any) {
```

Comment:
Can we move these funtion (getUserInfo and getImageUrl) to some other file.. its doesn't look like an appropriate fit inside Authentication.tsx file or do we need to rename this file ?